.gitignore

```
@@ -1,2 +1,8 @@
 doc/build
 Manifest.toml
+*.swp
+.vscode/settings.json
+docs/build/
+docs/build/assets/
+Project.toml
+*.html
```

removing `Project.toml` is a bad idea.
If we want to ignore VScode files, I'd rather remove the whole folder.
Ignoring `docs/build` makes sense, this rule should cover all subfolders and generated files.

```suggestion
*.swp
.vscode
docs/build/
```